modin/core/execution/ray/common/engine_wrapper.py

```diff
@@ -18,9 +18,13 @@
 """
 
 import asyncio
+from os import environ
```

Comment:
Let's use the following import (and further like `os.environ`)
```suggestion
import os
```